lib/MetaCPAN/Web/Controller/Root.pm

```diff
@@ -83,6 +83,16 @@ sub robots : Path("robots.txt") : Args(0) {
     $c->stash( { template => 'robots.txt' } );
 }
 
+sub service_worker : Path("service_worker") : Args(0) {
+    my ( $self, $c ) = @_;
+    $c->res->content_type('application/javascript');
+    $c->stash( {
+        version      => 'v1.0',
```

Comment:
This should really come from a config file or env variable - couldn't figure out if/where a application/release version is available.